### DISCUSSION TITLE
quick terminal support splits, add on website

### DISCUSSION THREAD

**Discussion by AnirudhG07** [ghostty-org/ghostty#3318] [Feature Requests, Ideas] [answered]
Hey! Thanks for the amazing ghostty!
I was reading the docs and I never saw any place where it states that quick terminal supports splitting into panes, it does not support tabs which is present, but some people who want to use different tabs can atleast benefit from the fact that you know open splits!(kind of like a reminder) 
Is it possible if I can add it [here](https://ghostty.org/docs/config/keybind/reference#toggle_quick_terminal) in the documentations for the website??
Thanks!

**Comment by mitchellh** [comment, marked as answer]
#3518
